samples/crypto/aes_cbc/CMakeLists.txt

```
@@ -10,7 +10,5 @@ find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
 
 project(aes_cbc)
 
-target_include_directories(app PUBLIC ../crypto_common)
```

I assume this would also have solved the issue:
```suggestion
target_include_directories(app PUBLIC ${ZEPHYR_NRF_MODULE_DIR}/samples/crypto/crypto_common)
```

But i'm also fine with the set of changes in this PR, as it should be very obvious that most of this print code can be safely removed in user's final application, whereas it is a bit harder to determine if:
```
#include <crypto_common.h> 
```
is safe to remove, unless the user actually inspects that header in details.